### ISSUE TITLE
Feature: Gathering statistics

### ISSUE THREAD

**Issue by cairnsh** [cairnsh/roadnetwork#1]
The model should gather statistics about:

- lengths of lines & probability distribution of length × √N
- angle of connections
- number of components (is there really almost always 1?)

**Comment by cairnsh** [comment]
I forgot this but it already gathers information about line lengths and writes them to the file `len.dat`. The n-th entry is the total length of lines added at step n.

**Comment by at82** [comment]
So we can easily make a histogram of length× √N

**Comment by at82** [comment]
Is the n-th entry the  total length of lines added up to step n? Which means the length for the line added at step n is the n-th entry minus the (n-1)-th entry?

**Comment by cairnsh** [comment]
No, I think it's the sum of the lengths of the line or lines added at step n (there might be more than one)
This is the part that makes the file:

```
        dist,shortest = netx.bidirectional_dijkstra(self.componentgraph, "x", "y")
        self.llengths.write("%f\n" % dist)
        self.llengths.flush()
```
